tests/data_engine/annotation_import/test_cvat_video.py

```
@@ -0,0 +1,318 @@
+    ann.video_height = 0
+    ann.filename = "missing.mp4"
+    dp.metadata["ann"] = MetadataAnnotations(datapoint=dp, field="ann", annotations=[ann])
+    qr = _make_qr(ds, [dp], ann_field="ann")
+
+    def _mock_download_files(self, target_dir, *args, **kwargs):
+        target_dir.mkdir(parents=True, exist_ok=True)
+        return target_dir
+
+    monkeypatch.setattr(QueryResult, "download_files", _mock_download_files)
+
+    with pytest.raises(FileNotFoundError, match="missing.mp4"):
```

Ruff (0.15.9)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)